Update ch04-02-references-and-borrowing.md

beling opened the pull request:
początek tłumaczenia
PS. Czy mogę prosić o możliwość zapisywania do repozytorium?

m4sk1n commented:
Mógłby ktoś zobaczyć, czy nie zrobiłem głupiego błędu przy rozwiązywaniu merge conflitcs?

paytchoo commented:
Szacun. Po długiej przerwie wreszcie mogę wrócić do roboty.

beling committed:
Update ch04-02-references-and-borrowing.md

początek tłumaczenia
PS. Czy mogę prosić o możliwość zapisywania do repozytorium?
